lib/calendar.ts

```diff
@@ -1422,6 +1482,12 @@ class IndianHelper extends HelperBase {
   maximumMonthLength(calendarDate: CalendarYM) {
     return this.getMonthInfo(calendarDate).length;
   }
+  maxLengthOfMonthCodeInAnyYear(monthCode: string) {
+    const month = +ES.Call(StringPrototypeSlice, monthCode, [1]);
```

Comment:
Conversion of monthCode => number is repeated in many places. Maybe this should be DRY-ed?